azure-pipelines-full-tests.yml

```diff
@@ -140,3 +140,37 @@ jobs:
       instance_idx: '$(Instance_idx)'
       fullTest: true
       jobName: 'FullTest'
+
+- job: NotifyCIErrors
+  dependsOn:
+    - AutomationTest20200901
+    - AutomationTest20190301
+    - AutomationTest20180301
+    - AutomationFullTestPython39ProfileLatest
+    - AutomationFullTestPython310ProfileLatest
+  condition: and(failed(), in(variables['Build.Reason'], 'BatchedCI'))
+  displayName: Notify CI Errors
+  pool:
+    name: ${{ variables.ubuntu_pool }}
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.10'
+    inputs:
+      versionSpec: 3.10
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: 'Azure CLI'
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        pip install requests
+        teams_api_url=$(az keyvault secret show --vault-name kv-azuresdk --name azure-cli-teams-bot-api-url --query value -otsv)
+        teams_api_key=$(az keyvault secret show --vault-name kv-azuresdk --name azure-cli-teams-bot-api-key --query value -otsv)
```

Comment:
I think we should get the vault-name and secret-name from the environment. For security concern.

Reply:
ADO will prevent pull request from fork repo trigger jobs that contain sensitive information.
https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#contributions-from-forks
![image](https://user-images.githubusercontent.com/18628534/231052388-681d513e-87c6-4a5e-b562-235ba7fd5d3c.png)